vllm/model_executor/layers/rotary_embedding/common.py

```diff
@@ -132,9 +132,14 @@ def __init__(
 
         self.apply_rotary_emb_flash_attn = None
         if find_spec("flash_attn") is not None:
-            from flash_attn.ops.triton.rotary import apply_rotary
-
-            self.apply_rotary_emb_flash_attn = apply_rotary
+            try:
+                from flash_attn.ops.triton.rotary import apply_rotary
+
+                self.apply_rotary_emb_flash_attn = apply_rotary
+            except Exception:
+                # If flash_attn is installed but mismatched with torch,
+                # fall back to the native path.
+                self.apply_rotary_emb_flash_attn = None
```

Comment:
Can you revert this unrelated change? If it is a real issue, we should address in a separate PR